iceoryx_examples/callbacks_in_c/ice_c_callbacks_subscriber.c

```diff
@@ -132,7 +132,7 @@ int main()
     iox_sub_options_t options;
     iox_sub_options_init(&options);
     options.historyRequest = 10U;
-    options.queueCapacity = 5U;
+    options.queueCapacity = 50U;
```

Comment:
You could think about updating the year in the copyright header. However, no blocker for me.